setup.py

```diff
@@ -182,6 +182,11 @@ def write_version(filename: str = os.path.join(*["airflow", "git_version"])):
     'sphinxcontrib-httpdomain>=1.7.0',
 ]
 docker = ['docker~=3.0']
+drill = [
+    'pydrill>=0.3.4',
+    'sqlalchemy-drill@'
+    'git+https://github.com/aksakalli/sqlalchemy-drill.git@master#egg=sqlalchemy-drill==2.22.0'
```

Comment:
I'm not sure if it's a good idea to introduce dependency in such a way.

Reply:
I need to use this repository, Unfortunately, this repository does not exist on `PYPI` PACKAGE.

Reply:
Then this PR can't be accepted in to Airflow.

Reply:
Because of using none PIPY pacakge?

Reply:
This is to ensure that each release/version can run stably & would not be broken because of dependency. 

Believe you can understand @blcksrx . I would suggest to close this PR first & re-open it once you address this essential 'issue'. Thanks for the contribution though, and we would be happy to take another look when you make it ready.

Reply:
You are Right. Unfortunately, the repo is not active and I' will close it. thanks.